wbooks-api/app/services/open_library.rb

```diff
@@ -0,0 +1,22 @@
+class OpenLibrary
+  def self.valid_isbn?(isbn)
+    (isbn.length == 10 || isbn.length == 13) && isbn.scan(/\D/).empty?
+  end
+
+  def self.search(isbn)
+    return {} unless valid_isbn?(isbn)
+
+    response = HTTParty.get("https://openlibrary.org/api/books?bibkeys=ISBN:#{isbn}"\
```

Comment:
Freeze string
![](https://pm1.narvii.com/6539/19b1e6ee48c9c45f4015855eeb24ab25d8672b25_hq.jpg)
https://freelancing-gods.com/2017/07/27/an-introduction-to-frozen-string-literals.html

Reply:
I would add an environment variable for `https://openlibrary.org/api`